lib/db.js

```diff
@@ -238,6 +238,12 @@ module.exports = (
 
     return P.all(promises)
       .spread((mysqlSessionTokens, redisTokens) => {
+        if (config.tokenLifetimes.sessionTokenWithoutDevice) {
+          // Filter out any expired sessions
+          mysqlSessionTokens = mysqlSessionTokens.filter(sessionToken =>
+            sessionToken.createdAt > Date.now() - config.tokenLifetimes.sessionTokenWithoutDevice
```

Comment:
Should this also check that the sessionToken does not have a corresponding device, or is that covered elsewhere?